.circleci/config.yml

```diff
@@ -1170,7 +1170,9 @@ jobs:
 
   lint-python:
     docker:
-      - image: determinedai/cimg-base:stable
+      - image: cimg/go:1.16
```

Comment:
the `lint-python` job needs to run in the `cimg-base` docker image.  Only the `build-proto` is going to use the go image.